control/mpc_follower/src/mpc_follower_core.cpp

```diff
@@ -701,7 +716,10 @@ bool MPCFollower::executeOptimization(
 
   if (Uex->array().isNaN().any()) {
     RCLCPP_WARN_SKIPFIRST_THROTTLE(
-      get_logger(), *get_clock(), 1.0, "model Uex includes NaN, stop MPC. ");
+      get_logger(), *get_clock(), duration,
+      "model Uex includes NaN, "
+      "stop MPC."
+      " ");
```

Comment:
The indent is broken. Same for other throttle functions, too.

Reply:
ahr, I ran clang-format and didn't check how it formatted it. I concatenated the strings manually and now clang-formate preserves them. weird